### PR TITLE
Adds event model

### LINKED ISSUE #9: Event model missing
Hi, I'm trying to migrate an event konnector (Meetup) but the `event` model is missing from the lib.

I tried to look how those were made, but I'm quickly lost with the `cozyclient` API and didn't find any doc to give me hints.

Is someone already doing this? If not, I could try but some documentation on the client API would be more than welcome 😄 

Thanks

### PR DESCRIPTION
**Pull request by JbIPS**
First attempt at adding the model. I'm still testing it but this will let anyone have a look and maybe spot mistakes 😃 

Closes #9

### CODE REVIEW

File: models/event.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+const cozy = require('../cozyclient');
+const DOCTYPE = 'io.cozy.events';
+
+module.exports = {
+  displayName: 'Event',
+  doctype: DOCTYPE,
+  all (callback) {
+    cozy.data.defineIndex(DOCTYPE, [])
```
Comment (doubleface) [inline models/event.js]: Did you check it works? I think a defineIndex without attribute does not work to get all the elements. You can get an example of implementation [here](https://github.com/JbIPS/cozy-konnector-libs/blob/30b15f71b538e339f0d82fdf76c7a573f48dcf93/models/base_model.js#L9).

And better, you could directly use the baseModel and not have to reimplement everything.
You can use the baseModel like this : 

```javascript
const baseModel = require('./base_model')
module.exports = baseModel.createNew({
  name: DOCTYPE,
  // put your code here
})
```

**Comment by doubleface** [comment]
And @JbIPS, thanks for the work by the way :smile:

File: models/event.js (hunk 1 above)
Comment (JbIPS) [inline models/event.js]: I tried it with the fixture and it worked, but I'll try with your implementation 😉

Comment (doubleface) [inline models/event.js]: The cozy-client-js stub (which uses fixture file) does not use the parameters and then it is not a very good test in this case. This won't work when using the yarn dev command which uses the real implementation of cozy-client-js

**Comment by doubleface** [comment]
I'm sorry, I have to close this PR for now since it is not usable in a connector.

**Pull request closed without merging**